src/firstboot/firstboot.c

```diff
@@ -228,6 +229,19 @@ static int prompt_loop(const char *text, char **l, unsigned percentage, bool (*i
         }
 }
 
+static int should_configure(int dir_fd, const char *filename) {
+        int r;
+
+        assert(dir_fd >= 0);
+        assert(filename);
+
+        r = faccessat(dir_fd, filename, F_OK, AT_SYMLINK_NOFOLLOW);
+        if (r < 0 && errno != ENOENT)
+                return log_error_errno(errno, "Failed to access %s: %m", filename);
+
+        return r < 0 || arg_force;
```

Comment:
doesn't matter, but i'd format this like this:

```c
if (faccessat(…) < 0) {
        if (errno != ENOENT)
                return log_error_errno(errno, …);

        return true; /* missing */
}

return arg_force; /* exists, but it's ok if --force was given */
```

gets rid of one variable, does only one < 0 check...